app/controllers/contacts_controller.rb

```diff
@@ -37,7 +39,7 @@ class ContactsController < ApplicationController
   end
 
   private def new_profile_data_key_to_value(new_profile_data, profile_data_to_change)
-    if profile_data_to_change == 'country'
+    if ['country', 'country_iso2'].include?(profile_data_to_change)
```

Comment:
How can both variants be possible? Shouldn't you make sure that the frontend is consistent enough to focus on either one?

(Or am I correct in guessing that one is from legacy Ruby templates and the other is from React?)